### DISCUSSION TITLE
How to determine the matching version of Eclipse Equinox for Eclipse IDE for Embedded C/C++ Developers 2024-09?

### DISCUSSION THREAD

**Discussion by yaccwang** [eclipse-equinox/equinox#694] [Q&A]
Hello,

I am currently using the Eclipse IDE for Embedded C/C++ Developers with the following version information:

Version: 2024-09 (4.33.0)
Build ID: 20240905-0614
Could you please advise how I can determine which version of Eclipse Equinox is compatible with this version?

Thank you!

**Comment by merks** [comment]
Better to ask the consuming about that:

https://github.com/eclipse-embed-cdt/eclipse-plugins/discussions

You can look at what's in your installation via the Help -> About Eclipse

![image](https://github.com/user-attachments/assets/a3ffa05b-cfc1-45c7-b6c9-0cf11ef9ed7a)

**Comment by tjwatson** [comment]
I'm sure I am missing the motivation behind the question.  The clear answer to me is whatever version the Eclipse IDE for Embedded C/C++ includes in its dependency on Equinox.

Why are you asking?
